### DISCUSSION TITLE
How to install mpdf?

### DISCUSSION THREAD

**Discussion by sscherger** [mpdf/mpdf#2019] [General]
Hi,
sorry, shame on me, but I'm not able to get started with mpdf.
I wrote the software the whole company works with, and we still use FPDF and TCPDF, but I don't know how to add MPDF to our software.
We use PHP 8.X with MySQL and Apache on Windows.
So far I have loaded the 8.2.3 zip from Github and put this in a folder "mpdf". 
But what means that line?:
$ composer require mpdf/mpdf

And what about that line when trying to test the simple usage?:
require_once __DIR__ . '/vendor/autoload.php';

What means vendor? Where should this autoload.php to be found?

Sorry for that stupid questions here, but I see no other chance to get started with mpdf. Later on I would expect this would behave quite similiar to FPDF or TCPDF.

Thank you for your time if you could help me please!
Best Regards
Stefan

**Comment by finwe** [comment]
1) install this: https://getcomposer.org/doc/00-intro.md so that you can use the command `composer` in your commandline
2) run the command `composer require mpdf/mpdf` in the commandline, when in the root directory of your project
3) the command will create the `vendor/autoload.php` file which you will require where you want to use mPDF.
